docs/en/changes/changes.md

```diff
@@ -45,6 +45,7 @@
   - NGINX -> K8S_SERVICE
   - APISIX -> K8S_SERVICE
   - GENERAL -> APISIX
```

Comment:
@peachisai Please reference this to update your changelog as new features are included. And you need to resolve the conflicts.